EIPS/eip-99999.md

```diff
@@ -0,0 +1,119 @@
+---
+eip: 99999
+title: Remove legacy `deposit` and `eth1data` fields
+description: Remove `deposits` and `eth1_data` fields from `BeaconBlockBody` after EIP-6110 finalization
+author: Terence (@terencechain)
+discussions-to: <URL>
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5